lib/std/os/test.zig

```diff
@@ -1042,26 +1042,21 @@ test "access smoke test" {
 }
 
 test "timerfd" {
-    if (true) {
-        // https://github.com/ziglang/zig/issues/13721
-        return error.SkipZigTest;
-    }
-
     if (native_os != .linux)
         return error.SkipZigTest;
 
     const linux = os.linux;
     var tfd = try os.timerfd_create(linux.CLOCK.MONOTONIC, linux.TFD.CLOEXEC);
     defer os.close(tfd);
 
+    // Fire event after absolute time of 10_000_000ns = 10ms
```

Comment:
absolute time is relative to the clock used to create the timerfd: `CLOCK.MONOTONIC`, not relative to the `timerfd_settime`. To wait for 10ms, either add 10ms to the current `CLOCK.MONOTONIC` or use `timer_settime(0)` for an it_value relative to that call instead.

Reply:
relative sounds simpler to me, so I use that.